assignment2_exprob/Marker_id_pos[] markers